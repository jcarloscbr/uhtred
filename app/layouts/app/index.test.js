import React from 'react'
import { mount } from 'enzyme'
import AppLayout from 'layouts/app/index'
import renderer from 'react-test-renderer'

it('renders stock list', () => {
  const state = {
    stocks: [{ symbol: 'PETR4' }],
    series: [{ symbol: 'C' }],
    callOptions: [{ symbol: 'PETRB28' }]
  }

  const tree = renderer.create(
    <AppLayout stocks={state.stocks} series={state.series} callOptions={state.callOptions} />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
