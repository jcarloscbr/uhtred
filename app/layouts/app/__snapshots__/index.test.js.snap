// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stock list 1`] = `
<div
  className="AppLayout-root-1"
>
  <div
    className="AppLayout-appFrame-2"
  >
    <header
      className="MuiPaper-root-11 MuiPaper-shadow4-17 MuiAppBar-root-4 MuiAppBar-positionFixed-5 MuiAppBar-colorPrimary-9 mui-fixed HeaderToolbar-headerToolbar-3"
    >
      <div
        className="MuiToolbar-root-38 MuiToolbar-gutters-39"
      >
        <h2
          className="MuiTypography-root-40 MuiTypography-title-46 MuiTypography-colorInherit-59 MuiTypography-noWrap-56"
        >
          
        </h2>
      </div>
    </header>
    <div
      className="MuiDrawer-docked-65"
    >
      <div
        className="MuiPaper-root-11 MuiPaper-shadow0-13 MuiDrawer-paper-66 Sidebar-drawerPaper-64 MuiDrawer-paperAnchorDockedLeft-71"
      >
        <div
          className="Brand-drawerHeaderMixin-74 Brand-drawerHeader-75"
        >
          <h2
            className="MuiTypography-root-40 MuiTypography-title-46 MuiTypography-colorPrimary-60 MuiTypography-noWrap-56 Brand-drawerHeaderTitle-76"
          >
            Marketr
          </h2>
          <h3
            className="MuiTypography-root-40 MuiTypography-subheading-47 MuiTypography-colorPrimary-60 MuiTypography-noWrap-56"
          >
            About
          </h3>
        </div>
        <hr
          className="MuiDivider-root-77 MuiDivider-default-78"
        />
        <ul
          className="MuiList-root-82 MuiList-padding-83"
        >
          <li
            className="MuiButtonBase-root-96 MuiListItem-root-86 MuiListItem-gutters-93 MuiListItem-button-94 MuiListItem-default-89"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root-98"
            >
              <h3
                className="MuiTypography-root-40 MuiTypography-subheading-47 MuiListItemText-text-101"
              >
                PETR4
              </h3>
            </div>
            <span
              className="MuiTouchRipple-root-103"
            />
          </li>
        </ul>
      </div>
    </div>
    <main
      className="Main-content-110"
    >
      <header
        className="MuiPaper-root-11 MuiPaper-shadow4-17 MuiAppBar-root-4 MuiAppBar-positionStatic-7 MuiAppBar-colorPrimary-9"
      >
        <div
          className="MuiTabs-root-111"
        >
          <div
            className="MuiTabs-flexContainer-112"
          >
            <div
              className="MuiTabs-scrollingContainer-113 MuiTabs-fixed-114"
              onScroll={[Function]}
              role="tablist"
              style={
                Object {
                  "marginBottom": 0,
                }
              }
            >
              <div
                className="MuiTabs-flexContainer-112"
              >
                <button
                  aria-selected={true}
                  className="MuiButtonBase-root-96 MuiTab-root-118 MuiTab-rootInherit-126 MuiTab-rootInheritSelected-127"
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  role="tab"
                  style={undefined}
                  tabIndex={0}
                  type="button"
                >
                  <span
                    className="MuiTab-wrapper-130"
                  >
                    <div
                      className="MuiTab-labelContainer-131"
                    >
                      <span
                        className="MuiTab-label-132"
                      >
                        C
                      </span>
                    </div>
                  </span>
                  <span
                    className="MuiTouchRipple-root-103"
                  />
                </button>
              </div>
              <div
                className="MuiTabIndicator-root-134 MuiTabIndicator-colorAccent-135"
                style={
                  Object {
                    "left": 0,
                    "width": 0,
                  }
                }
              />
            </div>
          </div>
        </div>
      </header>
      <table
        className="MuiTable-root-137"
      >
        <thead
          className="MuiTableHead-root-138"
        >
          <tr
            className="MuiTableRow-root-139 MuiTableRow-head-140"
          >
            <th
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147 MuiTableCell-head-146"
            >
              Trades
            </th>
            <th
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147 MuiTableCell-head-146"
            >
              Serie
            </th>
            <th
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147 MuiTableCell-head-146"
            >
              Strike
            </th>
            <th
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147 MuiTableCell-head-146"
            >
              Price
            </th>
          </tr>
        </thead>
        <tbody
          className="MuiTableBody-root-151"
        >
          <tr
            className="MuiTableRow-root-139"
          >
            <td
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147"
            />
            <td
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147"
            >
              PETRB28
            </td>
            <td
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147"
            />
            <td
              className="MuiTableCell-root-144 MuiTableCell-paddingDefault-147"
            />
          </tr>
        </tbody>
      </table>
    </main>
  </div>
</div>
`;
